rec_to_nwb/processing/tools/get_times_period_multiplier.py

```diff
@@ -0,0 +1,2 @@
+def get_times_period_multiplier(metadata):
+    return metadata.get('times_period_multiplier', 1.5)
```

Comment:
Does that handle potential problems with data type? e.g. string/datetime instead of decimal/number.